deker/integrity.py

```diff
@@ -170,38 +170,43 @@ def check_arrays_locks(self, collection: Collection) -> None:
         if self.stop_on_error and self.errors:
             raise DekerIntegrityError(self._parse_errors())
 
-    def check(self, collection: Collection) -> None:
-        """Check if Arrays or VArray in Collection are valid.
+    def _check_v_arrays(self, collection: Collection, v_arrays: Union[ArrayManager, Optional[VArrayManager]]) -> None:
```

Comment:
If ArrayManger is also can be passed, it's better to rename argument as you have to deal with both types of managers